src/Components/ConfirmEmailPage.jsx

```diff
@@ -0,0 +1,15 @@
+import "./confirmEmail.scss";
+import { useSearchParams } from "react-router-dom";
+import "../lib/appwrite/services/auth.service"
+
+export default function ConfirmEmail() {
+  const [params] = useSearchParams();
+  const userId = params.get("userId");
+  console.log(userId);
+  const secret = params.get("secret");
+  console.log(secret);
+
+    const verifyEmailHandler = await createUserVerification();
```

Comment:
you dont need this variable instead do it inside a useEffect
```suggestion
useEffect(()=>{
await createUserVerification();
},[]) 
```

Reply:
fixed

Reply:
fixed